docs/matlab/Getting_Started.m

```diff
@@ -0,0 +1,250 @@
+% environment.
+% 
+% We start by configuring the Python environment. You can do so by following 
+% the guidelines presented in the MATLAB® documentation: <https://www.mathworks.com/help/matlab/matlab_external/install-supported-python-implementation.html?searchHighlight=python&s_tid=srchtitle_python_4 
+% Configure Your System to Use Python - MATLAB® & Simulink (mathworks.com)>.
+% 
+% Once the Python environment is configured, RocketPy needs to installed using 
+% |pip| as outlined in RocketPy's documentation: <https://docs.rocketpy.org/en/latest/user/installation.html#quick-install-using-pip 
+% Installation — RocketPy documentation>.
+% 
+% Finally, all the prerequisites are complete and you can comeback to MATLAB®! 
+% We just need to set the execution mode as out of process and start working. 
+% MATLAB® can run Python scripts and functions in a separate process. Running 
+% Python in a separate process enables you to:
+%% 
+% * Use some third-party libraries in the Python code that are not compatible 
+% with MATLAB®.
+% * Isolate the MATLAB® process from crashes in the Python code.
+
+pyenv('ExecutionMode','OutOfProcess');
+%% 
+% Now, we will go through a simplified rocket trajectory simulation to get you 
+% started. Let's start by importing the rocketpy module.
+
+rocketpy = py.importlib.import_module('rocketpy');
+%% Setting Up a Simulation
+% Creating an Environment for Spaceport America
+
+% rocketpy.Environment(railLength, latitude, longitude, elevation);
+Env = rocketpy.Environment(pyargs(...
+    'railLength', 5.2, ...
+    'latitude', 32.990254, ...
+    'longitude',-106.974998, ...
+    'elevation', 1400 ...
+));
+%% 
+% To get weather data from the GFS forecast, available online, we run the following 
+% lines.
+% 
+% First, we set tomorrow's date.
+
+Tomorrow = datetime('tomorrow');
+Env.setDate({int32(Tomorrow.Year), int32(Tomorrow.Month), int32(Tomorrow.Day), int32(12)}) % Hour given in UTC time (noon UTC)
+%% 
+% Now, we tell our Environment object to retrieve a weather forecast for our 
+% specified location and date using GFS:
+
+Env.setAtmosphericModel(pyargs( ...
+    'type', "Forecast", ...
+    'file', "GFS" ...
+))
+%% 
+% We can see what the weather will look like by calling the info method!
+
+Env.info()
+%% 
+% Plots will open in a separate window, so be sure to run this last cell to 
+% see them!
+%% Creating a Motor
+% A solid rocket motor is used in this case. To create a motor, the SolidMotor 
+% class is used and the required arguments are given.
+% 
+% The SolidMotor class requires the user to have a thrust curve ready. This 
+% can come either from a .eng file for a commercial motor, such as below, or a 
+% .csv file from a static test measurement.
+% 
+% Besides the thrust curve, other parameters such as grain properties and nozzle 
+% dimensions must also be given.
+
+Pro75M1670 = rocketpy.SolidMotor(pyargs( ...
+    'thrustSource', "../../data/motors/Cesaroni_M1670.eng", ...
```

Comment:
I may be worth mentioning that for these file paths to work, the Matlab working directory has to be the matlab folder inside the RocketPy repository, or else the files cannot be found.